src/formatting/unused_parents.rs

```diff
@@ -340,7 +486,6 @@ y = 2
 ",
             str![[r#"
 # leading comment
-[parent]
 [parent.child]
```

Comment:
Isn't a comment like this part of `parent`, just like a comment on the same line?